recipe/build.sh

```diff
@@ -10,3 +10,5 @@ fi
 make
 make check
 make install
+
+mv ${PREFIX}/lib/*/include/* ${PREFIX}/include
```

Comment:
This strikes me as the wrong way to do this because it doesn't fix the *.la file(which doesn't seem to need it). I think the correct fix is to find the proper configure option.